src/android/patches/makeStringsPatch.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function makeStringsPatch(params) {
+  const patch = Object.keys(params).map(key =>
+    `    <string moduleConfig="true" name="${key}">${params[key]}</string>`
```

Comment:
> Which issues would we have if we use strings.xml? Which benefits rnpm.xml brings to the table?

Please read my previous comments, I am not going to keep that discussion all over again ;)